packages/govern-contract-utils/contracts/protocol/IArbitrable.sol

```diff
@@ -2,14 +2,17 @@
  * SPDX-License-Identifier:    MIT
  */
 
-// From https://github.com/aragon/aragon-court/blob/master/contracts/arbitration/IArbitrable.sol
+// From https://github.com/aragon/protocol/blob/25dafb59229e720c6e31e51830788185e43ad15e/packages/evm/contracts/arbitration/IArbitrable.sol
 
 pragma solidity ^0.6.8;
 
 import "./IArbitrator.sol";
 
+/**
+* @dev The Arbitrable instances actually don't require to follow any specific interface.
+*      Note that this is actually optional, although it does allow the Protocol to at least have a way to identify a specific set of instances.
+*/
 abstract contract IArbitrable {
-    bytes4 internal constant ARBITRABLE_INTERFACE_ID = bytes4(0x88f3ee69);
     uint256 internal constant ALLOW_RULING = 4;
```

Comment:
Perhaps we should move this constant into `GovernQueue`? Arbitrables choose this value themselves.